pkg/kubelet/kubelet.go

```diff
@@ -220,10 +220,12 @@ var (
 	// ContainerLogsDir can be overwritten for testing usage
 	ContainerLogsDir = DefaultContainerLogsDir
 	etcHostsPath     = getContainerEtcHostsPath()
+
+	goos = sysruntime.GOOS
```

Comment:
nit: I'd prefer to keep this inline, rather than assigning to a variable. It's just a constant, no cost to calling it multiple times.

Reply:
It is mainly for unit tests. If I mutate runtime.GOOS during unit tests, then it might affect other unit tests running in parallel. Let me verify this.

Reply:
Looks like runtime.GOOS is read-only. So I won't be able to write unit tests for this. WDYT?

```
./kubelet_test.go:2753:19: cannot assign to runtime.GOOS (neither addressable nor a map index expression)
```

Reply:
Ah, makes sense. Can you add a comment then? Something like `// exposed for testing`

Reply:
Added the comment. I'm not sure if there's any security risk by switching between `runtime.GOOS` constant and `goos` variable